LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2024 OpenVoiceOS
```

Comment:
Not sure if you need to contact the advocate, but I would imagine the copyright would extend earlier than 2024?

Reply:
we need to fix this org wide, technically the OVOS Foundation didnt exist before 2024, so my understanding is that we only can assign the copyright to it now, by default i currently own it (in repos i created at least)

it's part of the technicalities we are handling now that the foundation has been formed